src/NzbDrone.Common/Processes/ProcessProvider.cs

```diff
@@ -361,9 +353,19 @@ private List<Process> GetProcessesByName(string name)
             return processes;
         }
 
-        private string GetMonoArgs(string path, string args)
+        private (string Path, string Args) GetPathAndArgs(string path, string args)
         {
-            return string.Format("--debug {0} {1}", path, args);
+            if (PlatformInfo.IsMono && path.EndsWith(".exe", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
More cases like these are a concern to me, I'm not sure it lead to the path + args we currently have, but it makes that approach make more sense.